server/app.js

```diff
@@ -8,15 +8,21 @@ import { Model } from 'objection';
 import swaggerDocument from '@/swagger.json';
 import Routes from '@routes/v1';
 import config from '@config';
+import path from 'path';
 
 import 'express-jsend';
 
 export const app = express();
+app.use(express.static('views/assets'));
```

Comment:
Please why do we have views all of a sudden ?

Reply:
removed